eng/scripts/Invoke-DependencyCheck.ps1

```diff
@@ -0,0 +1,77 @@
+Param(
+    [string] $newPackageDirectory
+)
+
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+
+$ignoreCheck = "sdk/synapse/azartifacts", "sdk/template/aztemplate", "sdk/azcore", "sdk/azidentity"
+
+$sdks = Get-AllPackageInfoFromRepo
+
+$sdkRoot = Join-Path $RepoRoot "sdk"
```

Comment:
I would avoid this and just pass the "sdk" as part of the parameter and avoid needing to resolve full paths.

Reply:
Fixed.